analyzers/src/SonarAnalyzer.VisualBasic/Rules/MethodParameterUnused.cs

```diff
@@ -67,7 +67,6 @@ public sealed class MethodParameterUnused : MethodParameterUnusedBase
                     // Bail-out if this is not a method we want to report on (only based on symbols checks)
                     var methodSymbol = c.SemanticModel.GetDeclaredSymbol(methodBlock);
                     if (methodSymbol == null
-                        || methodSymbol.IsAbstract
```

Comment:
this seems like a change in bevahior? is it intended? shouldn't a test be added for abstract methods?

Reply:
It's for coverage. This condition is unreachable because of syntax checks above. And there's case in UTs for it as well (`MustOverride`).